var/spack/repos/builtin/packages/dedisp/package.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+from spack.package import *
+
+class Dedisp(MakefilePackage, CudaPackage):
+    """GPU-based dedispersion package."""
+
+    homepage = "https://github.com/ajameson/dedisp"
+
+    git = "https://github.com/ajameson/dedisp.git"
+    version("1.0", branch="master")
```

Comment:
I think it is better in this case to list this as version `master` to avoid future confusion.
```suggestion
    version("master", branch="master", preferred=True)
```
See e.g. `flexi`. https://github.com/spack/spack/blob/1ac2ee8043ab802ff5c9161ac35c269f22df3c34/var/spack/repos/builtin/packages/flexi/package.py#L18

You'll see in `flexi` that you can also use specific commits with a calendar version for additional stability in versions.